README.md

```diff
@@ -38,6 +38,26 @@ Usage questions can be posted on the [Julia Discourse forum][discourse-tag-url]
 
 Contributions are very welcome, as are feature requests and suggestions. Please open an [issue][issues-url] if you encounter any problems. The [contributing page][contrib-url] has a few guidelines that should be followed when opening pull requests and contributing code.
 
+## Related packages
+
+There are several packages that extend Documenter in different ways. The JuliaDocs organization maintains:
+
+* [DocumenterTools.jl](https://github.com/JuliaDocs/DocumenterTools.jl)
+* [DocumenterMarkdown.jl](https://github.com/JuliaDocs/DocumenterMarkdown.jl)
+
+Other third-party packages that can be combined with Documenter include:
+
+* [Literate.jl](https://github.com/fredrikekre/Literate.jl)
```

Comment:
alphabetic order maybe?

Reply:
Now that the maintainers have added so many other interesting packages I'll leave it to them to decide the order.

I was right that there were others that I didn't know about and am glad to learn about them here!